memphis/functions.py

```diff
@@ -5,7 +5,8 @@
 def create_function(
     event,
     event_handler: callable,
-    use_async: bool = False
+    use_async: bool = False,
```

Comment:
readme is missing, and make sure the user should not handle the json in the return statement as well. Like mentioned in the Nodejs comments

Reply:
readme was missing, fixed that, and then the as_json I renamed to as_dict because that's more pythonic I think.

Function was already doing the conversion for the user, just wasn't documented (for both repos).